pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -186,29 +193,28 @@ func (pl *DefaultPreemption) getOffsetAndNumCandidates(numNodes int32) (int32, i
 
 // FindCandidates calculates a slice of preemption candidates.
 // Each candidate is executable to make the given <pod> schedulable.
-func (pl *DefaultPreemption) FindCandidates(ctx context.Context, state *framework.CycleState, pod *v1.Pod, m framework.NodeToStatusMap) ([]Candidate, error) {
+func (pl *DefaultPreemption) FindCandidates(ctx context.Context, state *framework.CycleState, pod *v1.Pod, m framework.NodeToStatusMap) ([]Candidate, int, framework.NodeToStatusMap, error) {
```

Comment:
It seems in all return statements, the 2nd return parameter always equals to len(NodeToStatusMap), can you confirm? If so, we can remove the 2nd return param.

Reply:
Great suggestion! they should be equal